spec/controllers/api/v2/application_controller_spec.rb

```diff
@@ -41,5 +42,33 @@
           .to eq('Token realm="Application"')
       end
     end
+
+    context 'manage course api integration' do
```

Comment:
Is "manage course api integration" a description of the state of the system? Sounds like what we're testing, so `describe` would be more appropriate.